doc/nrf/links.txt

```diff
@@ -94,8 +94,8 @@
 .. _`SMP over Bluetooth`: https://github.com/apache/mynewt-mcumgr/blob/master/transport/smp-bluetooth.md
 
 .. _`Matter GitHub repository`: https://github.com/project-chip/connectedhomeip
-.. _`Android CHIPTool`: https://github.com/nrfconnect/sdk-connectedhomeip/tree/bb3c7570/src/android/CHIPTool
-.. _`other controller setups`: https://github.com/nrfconnect/sdk-connectedhomeip/tree/bb3c7570/src/controller
+.. _`Android CHIPTool`: https://github.com/nrfconnect/sdk-connectedhomeip/tree/708685f4/src/android/CHIPTool
```

Comment:
It's supposed to be hash in sdk-connectedhomeip, not upstream.

Reply:
Right, will update it after merging new SHA